PowerEditor/installer/nativeLang/french.xml

```diff
@@ -672,6 +668,12 @@ The comments are here for explanation, it's not necessary to translate them.
                     <Item id="44107" name="Basculer vers Dossier en tant qu'Espace de travail"/>
                     <Item id="44109" name="Basculer vers Liste des documents"/>
                     <Item id="44108" name="Basculer vers Liste des fonctions"/>
+                    <Item id="44110" name="Enlever la couleur d'onglet"/>
+                    <Item id="44111" name="Appliquer la couleur d'onglet 1"/>
+                    <Item id="44112" name="Appliquer la couleur d'onglet 2"/>
+                    <Item id="44113" name="Appliquer la couleur d'onglet 3"/>
+                    <Item id="44114" name="Appliquer la couleur d'onglet 4"/>
+                    <Item id="44115" name="Appliquer la couleur d'onglet 5"/>
```

Comment:
Le texte original est "Apply Color N", sous entendu "Apply Color N to the current tab".
La traduction que tu as fourni est donc assez ambiguë.

Reply:
C'est à partir de la ligne 292 pour les "Apply Color N", alors que les "Apply Tab Colour N" sont bien à partir de la ligne 672 ^^'

Je viens bien de vérifier sur le fichier english.xml du master :
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/installer/nativeLang/english.xml#L292
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/installer/nativeLang/english.xml#L672

Reply:
@Mateos81 
Je pensais que cela prête à la confusion. C'est OK en relisant la traduction.